setuptools/command/bdist_wheel.py

```diff
@@ -369,7 +369,7 @@ def get_tag(self) -> tuple[str, str, str]:
             )
         return tag
 
-    def run(self) -> None:
+    def run(self):
```

Comment:
I opted out of annotating this specific `run` in this PR for the same reason I didn't do `initialize_options`.

(not I'm commenting on a specific commit, this PR doesn't remove any return annotation)